src/response_terminal.py

```diff
@@ -639,40 +756,46 @@ def get_response_in_terminal_mode(message) -> str:
             # if msg > '6' or msg < '1':
             #     return 'no such command'
             # else:
-                return function_developing()
-            # TO-DO
-            # elif msg == 'setup':
+            return function_developing()
+        # TO-DO
```

Comment:
This is the first point.
This function is still developing, so please change it back.